english/new-movie/new-movie.md

```diff
@@ -27,3 +27,11 @@ Clues, clues, everywhere clues that don't connect!
 
 Dammit! No way you're going to miss this movie, it's been years you've been waiting for it. Time to run!
 
+Do you:
+
+[run left?](../super-man/save-the-word.md)
+
+[run right?](../suprise/suprise.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.